tests/Microsoft.Identity.Web.Test/AzureIdentityForKubernetesClientAssertionTests.cs

```diff
@@ -0,0 +1,69 @@
+using System;
```

Comment:
license header info is missing...it should be added by default when you create a class now :(

Reply:
fixed